src/app/components/SideBar.tsx

```diff
@@ -7,24 +7,28 @@
  */
 
 import * as m from 'mithril';
+import AppState from '../models/AppState';
 import NavItem from './NavItem';
 
 export default class SideBar implements m.Component {
+    class: string;
+    showSidebar: boolean;
+
+    constructor(vnode: any) {
+        this.class = vnode.attrs.class || '';
+        this.showSidebar = AppState.showSidebar;
+    }
+
     view() {
         return (
-            <div class="container-fluid">
-                <div id="sidebar-wrapper">
-                    <ul class="sidebar-nav">
-                        <li class="sidebar-brand">
-                            <ul class="mr-auto list-unstyled components">
-                                <NavItem href="/logs" name="Logs" />
-                                <NavItem href="/runs" name="Runs" />
-                                <NavItem href="/logs/create" name="Create new log" />
-                                <NavItem href="/runs/create" name="Create new run" />
-                            </ul>
-                        </li>
-                    </ul>
-                </div>
-            </div>);
+            // Sidebar
+            <nav class={`jf-sidebar ${this.class} ${AppState.showSidebar ? '' : 'jf-sidebar-active'}`}>
```

Comment:
the `jf-sidebar-active class` is not behaving correctly, see screenshot for clarification. 
![image](https://user-images.githubusercontent.com/28677235/47436286-7c044e00-d7a6-11e8-8d2a-b07a92d3a0ef.png)
sidebar has an active state while it is not being displayed

Reply:
`.jf-content-active`'s values are inverted via media queries below 768 pixels. This will make sure the sidebar is shown by default on screens with width > 768px and hidden by default on width <= 768px.